src/test/resources/push-to-bitbucket.sh

```diff
@@ -0,0 +1,9 @@
+git checkout master
+git remote -v
+git --version
+REMOTE=`git remote`
+git remote set-url $REMOTE http://admin:admin@localhost:7990/bitbucket/scm/project_1/$REMOTE.git
```

Comment:
Slight preference to do this by setting `GIT_ASKPASS` because that is more realistic to how it would be done for real.